gradle/libs.versions.toml

```
@@ -0,0 +1,113 @@
+[versions]
+
+kotlin = "1.8.10"
+kotlin-plugin = "213-1.8.10-release-430-IJ6777.52"
+kotlinx-coroutines = "1.6.3"
+kotlinx-html = "0.7.5"
+jsoup = "1.15.3"
+idea = "213.6777.52"
+
+jackson = "2.12.7" # jackson 2.13.X does not support kotlin language version 1.4, check before updating
+jacksonDatabind = "2.12.7.1" # fixes CVE-2022-42003
+
+freemarker = "2.3.31"
+
+jetbrainsMarkdown = "0.3.1"
+
+soywiz-korte = "2.7.0"
+
+apache-maven = "3.5.0"
+apache-mavenArtifact = "3.8.5"
+apache-mavenArchiver = "2.5"
+apache-mavenPluginTools = "3.5.2"
+
+eclipse-jgit = "5.12.0.202106070339-r"
+
+## test dependency versions ##
+junit = "5.9.2"
+assertk = "0.25"
+
+## Gradle plugins ##
+gradlePlugin-shadow = "7.1.2"
+gradlePlugin-binaryCompatibilityValidator = "0.12.1"
+gradlePlugin-nexusPublish = "1.1.0"
+gradlePlugin-dokka = "1.7.10"
+gradlePlugin-gradlePluginPublish = "0.20.0"
+gradlePlugin-gradle = "4.0.1"
+
+## NPM ##
+node = "16.13.0"
+
+
+[libraries]
+
+eclipse-jgit = { module = "org.eclipse.jgit:org.eclipse.jgit", version.ref = "eclipse-jgit" }
+freemarker = { module = "org.freemarker:freemarker", version.ref = "freemarker" }
+jetbrainsIntelliJ-core = { module = "com.jetbrains.intellij.idea:intellij-core", version.ref = "idea" }
+jetbrainsIntelliJ-jpsStandalone = { module = "com.jetbrains.intellij.idea:jps-standalone", version.ref = "idea" }
+jetbrainsMarkdown = { module = "org.jetbrains:markdown", version.ref = "jetbrainsMarkdown" }
+jsoup = { module = "org.jsoup:jsoup", version.ref = "jsoup" }
+soywiz-korte = { module = "com.soywiz.korlibs.korte:korte", version.ref = "soywiz-korte" }
+
+## Kotlin libs ##
+kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
+
+kotlin-compiler = { module = "org.jetbrains.kotlin:kotlin-compiler", version.ref = "kotlin" }
+kotlin-idea = { module = "org.jetbrains.kotlin:idea", version.ref = "kotlin" }
+kotlin-common = { module = "org.jetbrains.kotlin:common", version.ref = "kotlin" }
+kotlin-core = { module = "org.jetbrains.kotlin:core", version.ref = "kotlin" }
+kotlin-native = { module = "org.jetbrains.kotlin:native", version.ref = "kotlin" }
+
+kotlinx-cli = { module = "org.jetbrains.kotlinx:kotlinx-cli", version = "0.3.4" }
+kotlinx-html = { module = "org.jetbrains.kotlinx:kotlinx-html", version.ref = "kotlinx-html" }
```

Doesn't seem to fail the tests (yet), but maybe it's worth using `*-jvm` as before, to avoid possible problems. If a problem with it surfaces, I don't want to have to roll back the whole PR, so imo it'd be nice to keep the artifacts the same as before, and any additional improvements separately

btw, the cli version 0.3.4 could be extracted into a variable, probably forgotten?

```suggestion
kotlinx-cli = { module = "org.jetbrains.kotlinx:kotlinx-cli-jvm", version = "0.3.4" }
kotlinx-html = { module = "org.jetbrains.kotlinx:kotlinx-html-jvm", version.ref = "kotlinx-html" }
```